student.rb

```diff
@@ -1,9 +1,17 @@
 class Student < Person
-  attr_reader :classroom
+  attr_accessor :classroom
```

Comment:
###  OPTIONAL 
- [ ] Kindly consider updating `attr_accessor :classroom` to `attr_reader :classroom` for improved control. This adjustment will allow access to the classroom information without permitting direct modifications 👍🏿

Reply:
Also this is what I now have on line 2
  attr_reader :classroom


is it correct ma? @prolajumokeoni